### ISSUE TITLE
Stop the Damn CAP.

### ISSUE THREAD

**Issue by SnesLover22** [developer-lfierrro743/Project-EaglercraftY-1.9-version#13]
OK, dude. stop with the hoax. This is all fake, as a programmer myself, I can say that this HTML code does NOTHING for the game. Lax1dude had a lot of work to do with MULTIPLE FILES, not just one. That launcher code just launches the other files to work. Its not as easy as 123. There is no 1.8 code to be changed to 1.9 code (https://github.com/developer-lfierrro743/Project-EaglercraftY-1.9-version/issues/5#issuecomment-1426070373). You are BAD AT PHOTOSHOPPING stop now! the Eaglercraft community is literally laughing at you right now! Like literally none of that code relates to 1.8 lol, you stupid 9 year old. You don't even know how to code! I can bet my lucky mario socks on that, and more! You are a disgrace to think tricking all these people will work. Plus, someones working on 1.12 right now. ACTUAL 1.12 too. I know you're fake too since all you have is a DAMN JAR FILE in this repository.  AND THE CODE JUST HAS THE LETTER b IN IT! Get exposed, fool! GET EXPOSED. If you think you could even get away with this, HA screw you. I bet that on March 20th, you'll say theres been a delay, or maybe, on purpose, post the wrong HTML file. 

You disgrace.

**Comment by SnesLover22** [comment]
And another thing. Minecraft java edition uses Java, ok? Eaglercraft converts java into Javascript, yeah? OK good we're caught up now. So the thing is, browsers don't use Java. They use Javascript. So you can't INSTALL THE DAMN Java SE BINARY INTO A BROWSER. I am serious here. Stop. The. Cap.

**Comment by Ironbrodiet** [comment]
> one last remark from me: ITS HTML not HMTL, DUMMY.

lol

**Comment by mikelikesto** [comment]
> OK, dude. stop with the hoax. This is all fake, as a programmer myself, I can say that this HTML code does NOTHING for the game. Lax1dude had a lot of work to do with MULTIPLE FILES, not just one. That launcher code just launches the other files to work. Its not as easy as 123. There is no 1.8 code to be changed to 1.9 code ([#5 (comment)](https://github.com/developer-lfierrro743/Project-EaglercraftY-1.9-version/issues/5#issuecomment-1426070373)). You are BAD AT PHOTOSHOPPING stop now! the Eaglercraft community is literally laughing at you right now! Like literally none of that code relates to 1.8 lol, you stupid 9 year old. You don't even know how to code! I can bet my lucky mario socks on that, and more! You are a disgrace to think tricking all these people will work. Plus, someones working on 1.12 right now. ACTUAL 1.12 too. I know you're fake too since all you have is a DAMN JAR FILE in this repository. AND THE CODE JUST HAS THE LETTER b IN IT! Get exposed, fool! GET EXPOSED. If you think you could even get away with this, HA screw you. I bet that on March 20th, you'll say theres been a delay, or maybe, on purpose, post the wrong HTML file.
> 
> You disgrace.

Looks like some is pretending to know what they are talking about without fact checking first.

**Comment by SnesLover22** [comment]
> > OK, dude. stop with the hoax. This is all fake, as a programmer myself, I can say that this HTML code does NOTHING for the game. Lax1dude had a lot of work to do with MULTIPLE FILES, not just one. That launcher code just launches the other files to work. Its not as easy as 123. There is no 1.8 code to be changed to 1.9 code ([#5 (comment)](https://github.com/developer-lfierrro743/Project-EaglercraftY-1.9-version/issues/5#issuecomment-1426070373)). You are BAD AT PHOTOSHOPPING stop now! the Eaglercraft community is literally laughing at you right now! Like literally none of that code relates to 1.8 lol, you stupid 9 year old. You don't even know how to code! I can bet my lucky mario socks on that, and more! You are a disgrace to think tricking all these people will work. Plus, someones working on 1.12 right now. ACTUAL 1.12 too. I know you're fake too since all you have is a DAMN JAR FILE in this repository. AND THE CODE JUST HAS THE LETTER b IN IT! Get exposed, fool! GET EXPOSED. If you think you could even get away with this, HA screw you. I bet that on March 20th, you'll say theres been a delay, or maybe, on purpose, post the wrong HTML file.
> > You disgrace.
> 
> Looks like some is pretending to know what they are talking about without fact checking first.

HAHA you make me laugh (unless you're saying the guy making this so said "eaglercrafty" is fake than good job he is fake). I know a thing or two bout coding, and pretty much fluent in HTML. also he just has a .jar file in his repository with the letter b as his code. It is fake as hell. Check it, man. He is literally starting a shroud of turin kinda hoax here! He don't know what he even doing! Don't be ignorant or arrogant, see the facts.

**Comment by Ironbrodiet** [comment]
> @developer-lfierrro743 this dude is a disgrace bro he is a faker fake as hell he should just drop dead man like for real who tf would just start a hoax 
fr bro

**Comment by ImmP3rfect** [comment]
yea like bro

**Comment by SnesLover22** [comment]
yeah here's the repo for 1.12 he just started though: https://github.com/apersongithub/Eaglercraft-1.12-Project

**Comment by SnesLover22** [comment]
UPDATE: Just like I thought he extended the release date to March 30th so he can ACTUALLY get some work done, and I have a theory that this EaglerY he'll realease will just have some texture packs in it. SHAME ON YOU, SHAME!!

**Comment by mikelikesto** [comment]
> @developer-lfierrro743 
> 
>  this dude is a disgrace bro he is a faker fake as hell he should just drop dead man like for real who tf would just start a hoax
> 
> @SnesLover22   
> 
> is someone making 1.12 for real??

Bro really created a alt to support his argument

**Comment by SnesLover22** [comment]
um who made an alt me or someone else? I only have 2 or 3 accounts, all under sneslover22 or 23 or 24 lol

**Comment by SnesLover22** [comment]
> > @developer-lfierrro743
> > this dude is a disgrace bro he is a faker fake as hell he should just drop dead man like for real who tf would just start a hoax
> > @SnesLover22
> > is someone making 1.12 for real??
> 
> Bro really created a alt to support his argument

yeah thats not me bro lol I only have accounts under sneslover.

**Comment by SnesLover22** [comment]
> > @developer-lfierrro743
> > this dude is a disgrace bro he is a faker fake as hell he should just drop dead man like for real who tf would just start a hoax
> > @SnesLover22
> > is someone making 1.12 for real??
> 
> Bro really created a alt to support his argument

and you're following him so idk man its kinda sus

**Comment by dez1023** [comment]
> OK, dude. stop with the hoax. This is all fake, as a programmer myself, I can say that this HTML code does NOTHING for the game. Lax1dude had a lot of work to do with MULTIPLE FILES, not just one. That launcher code just launches the other files to work. Its not as easy as 123. There is no 1.8 code to be changed to 1.9 code ([#5 (comment)](https://github.com/developer-lfierrro743/Project-EaglercraftY-1.9-version/issues/5#issuecomment-1426070373)). You are BAD AT PHOTOSHOPPING stop now! the Eaglercraft community is literally laughing at you right now! Like literally none of that code relates to 1.8 lol, you stupid 9 year old. You don't even know how to code! I can bet my lucky mario socks on that, and more! You are a disgrace to think tricking all these people will work. Plus, someones working on 1.12 right now. ACTUAL 1.12 too. I know you're fake too since all you have is a DAMN JAR FILE in this repository. AND THE CODE JUST HAS THE LETTER b IN IT! Get exposed, fool! GET EXPOSED. If you think you could even get away with this, HA screw you. I bet that on March 20th, you'll say theres been a delay, or maybe, on purpose, post the wrong HTML file.
> 
> You disgrace.

cringe but true

**Comment by Ironbrodiet** [comment]
lol

**Comment by IceeFoox** [comment]
I dont agree that he is 9 years old. The reason why he's not, you don´t have proof, so do not assume his age.

**Comment by ImmP3rfect** [comment]
yea

**Comment by SnesLover22** [comment]
ok but they still lied about the project, and I myself will file a DMCA cause this is just sad.

**Comment by IceeFoox** [comment]
And you will also be lying because first of all you do not own Minecraft, neither are you an employee

**Comment by SnesLover22** [comment]
yeah but I can tell them. and look he just abandoned the project cause he doesn't know how to code! He never started! (he, they, it doesn't matter. they lied.)

**Comment by SnesLover22** [comment]
The eagler community is laughing at his attempts too cause its so fake! He used it to get views on his Youtube Channel

**Comment by IceeFoox** [comment]
Broooooo, they literally said its gonna be here on March 30th... Welp

**Comment by SnesLover22** [comment]
see they lied after seeing me bust them

**Comment by SnesLover22** [comment]
lied and left i mean

**Comment by developer-lfierrro743** [comment]
uh people THis was too much work only a team of developers 
(Senenarly the developer of Eaglercraft 1.9.4)
So this project ABANDONED 
(heres my real age 15 for real)

**Comment by SnesLover22** [comment]
bro you sound so fake! Also lax1dude worked on it by himself

**Comment by SnesLover22** [comment]
you can't even spell, man!
